sled-agent/src/long_running_tasks.rs

```diff
@@ -206,9 +207,10 @@ async fn spawn_bootstore_tasks(
     )
     .unwrap();
 
+    let log = log.new(o!("stage" => SWITCH_ZONE_INIT_STAGE));
```

Comment:
This means that all bootstore traffic related to trust quorum is going to be labelled "switch zone initialization" in perpetuity, right?